### ISSUE TITLE
Capture hypothesis id for all volunteers

### ISSUE THREAD

**Issue by yugen** [clingen-data-model/community-curation#14]
Acceptance Criteria:
* Collect hypothes.is username on volunteer application survey.
* Store hypothes.is username on volunteer account when created.

**Comment by courtneythaxton** [comment]
@yugen, would this be consider complete, as we just closed another ticket that indicated asking for a hypothes.is ID from volunteers

**Comment by yugen** [comment]
Good question, I didn't write the acceptance criteria in the initial description, but I've added it.  This one was to ask for hypothes.is username in the application survey.  You should be able to add a hypothes.is username in the application survey and it should get stored on the volunteer's account.

**Comment by courtneythaxton** [comment]
@yugen I tested this functionality by taking the survey.

1. the Hypothes.is ID was asked on the first page, however the user may not know from the prompt what this is for. Also it will only be needed for those that choose Baseline Annotation, so could we consider adding it after the point in which the volunteer chooses Baseline?

See image:
<img width="621" alt="Screen Shot 2020-06-02 at 11 23 36 PM" src="https://user-images.githubusercontent.com/32530005/83592428-531e7000-a528-11ea-9201-7e417631cb20.png">


2. I entered in all information on the survey that was indicated as required on the first page. When I hit the next button I was given this error:

<img width="893" alt="Screen Shot 2020-06-02 at 11 23 25 PM" src="https://user-images.githubusercontent.com/32530005/83592482-72b59880-a528-11ea-9b41-28c6e600cddf.png">
